From 40e57e131c6b53452b7f4d33a83dbc57ff20c884 Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Mon, 11 Oct 2021 17:08:04 +0300
Subject: [PATCH 11/14] s32: env: Set INITRD_HIGH before BL32+BL31+BL33 area

Updated the INITRD_HIGH macro to contain the address of
the end of the non-reserved memory area in DRAM Bank 1,
if booting with ATF.

Issue: ALB-7902
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/s32.h b/include/configs/s32.h
index 6b5be37eb7..235728d9f9 100644
--- a/include/configs/s32.h
+++ b/include/configs/s32.h
@@ -376,7 +376,7 @@
 
 /* Leave room for TF-A & OPTEE */
 #if defined(CONFIG_S32_ATF_BOOT_FLOW)
-#define INITRD_HIGH 0xFE7FFFFF
+#define INITRD_HIGH 0xFE1FFFFF
 #else
 #define INITRD_HIGH INITRD_HIGH_DEFAULT
 #endif
-- 
2.17.1

